src/seth/libexec/seth/seth-get-full-block-by-hash

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
These seem like basically dupes of the non-full counterparts. Maybe we can unify and make full/non-full an argument?

Reply:
They are essentially plumming, usually you just use `seth block` anyway, which now does take an option `--full`

Reply:
Right. I guess what I'm saying is that `seth-get-block-by-number` could get a `--full` argument too, since what's happening there and what's happening in `seth-get-full-block-by-number` is just copypasta.

Same for this file, although being just one line it's less important, but I still think it would be nice to reduce the number of files/commands in this case, since they can be cleanly thought of as just flags to a single argument.

Reply:
But I mean it's not a big deal, it just hurts my `a e s t h e t i c s` a bit.